### PR TITLE
Dev 3.0

### PR DESCRIPTION
**Pull request by andre91a**
Changes:

- Prerequisites: changed Neo4j and Linkurious Enterprise min versions

- Default styles: added edgeGroup

- Alerts: updated the guide and the alerts.cypher file for supporting the target

- Images: updated images where the old interface was displayed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # AML Dataset implementation guide
 
-### Prerequisites:
-- a  Neo4j 3.5.x server with read/write access
+### Prerequisites: 
+- a  Neo4j 4.2 or greater server with `admin` or `architect` access rights
 - APOC procedures are enabled in Neo4j (more information [here](https://neo4j.com/docs/labs/apoc/current/introduction/#installation))
-- a  Linkurious Enterprise 2.9.x server with admin access ([getting started guide](https://doc.linkurio.us/admin-manual/latest/getting-started/))
+- a Linkurious Enterprise 3.0 or greater with `admin` access rights ([getting started guide](https://doc.linkurio.us/admin-manual/latest/getting-started/))
```
Comment (fantandrea) [inline README.md]: I'll put 3.0.4 or greater (maybe higher is a better word?) since we changed the security of alerts

File: README.md [hunk 2]
```diff
@@ -124,22 +124,26 @@ Repeat this procedure for every alert in the file:
...
 1. On the *Alert dashboard*, click `CREATE NEW ALERT` (more information [here](https://doc.linkurio.us/user-manual/latest/alert-dashboard/)).
 
-2. Copy the query from the file and paste it in the `Query` field
+2. Fill the `Alert name` field with the value provided in the file
 
-4. Click on `ADD A COLUMN` and fill the fields with the values provided in the file \
-*NOTE: repeat this step if more columns are specified in the file* \
+3. Fill the `Description` field with the description provided in the file
+
+4. Turn on, if it's not, the `Enable` toggle \
```
Comment (fantandrea) [inline README.md:131]: I'll also add the step to share with all users otherwise the creator will be the only one able to work on it

**Review by fantandrea — CHANGES_REQUESTED** [review]
I've suggested few changes around the alert sharing. The rest is good

File: README.md (hunk 1 above)
Comment (andre91a) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (andre91a) [reply README.md:131]: done

**Pull request merged**